addons/website_slides/views/website_slides_templates_lesson.xml

```diff
@@ -42,7 +42,7 @@
             <div class="container o_wslides_lesson_main">
                 <div class="row">
                     <div t-attf-class="o_wslides_lesson_aside col-lg-3 #{'order-2' if slide.channel_id.channel_type == 'documentation' else ''}">
-                        <t t-if="slide.channel_id.channel_type == 'training'" t-call="website_slides.slide_aside_training"/>
+                        <t t-if="slide.channel_id.channel_type != 'documentation'" t-call="website_slides.slide_aside_training"/>
```

Comment:
does it change anything ? 
there are only two types of channel. 
You could either simplify in :
```suggestion
                        <t t-else="" t-call="website_slides.slide_aside_training"/>
```
and put it after 
`<t t-if="slide.channel_id.channel_type == 'documentation'" t-call="website_slides.slide_aside_documentation"/>`